test/inductor/test_codecache.py

```diff
@@ -1229,7 +1279,7 @@ def f(x, y, a, b):
             for k in global_stats.autotune_remote.cache.keys():
                 self.assertRegex(k, r"[0-9a-z]{52}\.py")
             for k in global_stats.triton.cache.keys():
-                self.assertRegex(k, r"triton:[0-9a-f]{64}::[0-9a-f]{64}:c10")
+                self.assertRegex(k, r"triton:[0-9a-f]{64}::[0-9a-f]{64}:c11")
```

Comment:
Do we want to compute this programmatically so if the version number changes again it automatically updates to the correct value?

Reply:
I can do that as a follow up, my fbcode setup is not working right now :(

lets unbreak first

Reply:
Maybe do it next time we update the value...